openmrs/apps/clinical/medication.json

```diff
@@ -25,7 +25,7 @@
             "doseUnits": "ml",
             "route": "Par voie orale"
           },
-          "Comprimé": {
+          "Comprimé(s)": {
```

Comment:
@prashantsable1991 This is not right, the dose form name is not Comprimé(s)

Reply:
@lauravignoli Thanks for raising.